cypress/integration/morecast-page.spec.ts

```diff
@@ -60,8 +65,7 @@ describe('MoreCast Page', () => {
       .first()
       .trigger('mousemove', { force: true, x: 2, y: 1 })
     cy.getByTestId('temp-rh-tooltip-text')
-      .should('contain', '3:00 pm, Thu, Oct 1st (PDT, UTC-7)')
-      .and('contain', 'Temp: - (°C)')
-      .and('contain', 'RH: 38 (%)')
+      .should('contain', 'Temp: - (°C)')
```

Comment:
Why have you removed the test for the time?

Reply:
I have to update it every time I update fixtures (which happens quite often) so I got annoyed and deleted it. Will bring it back

Reply:
Can you do a regex check that will always pass? You really don't care what the date is, just that there is a valid date.

Reply:
If it's too much trouble now - maybe throw TODO in there with the intention?

Reply:
Updated with Regex check 👍